folium/map.py

```diff
@@ -34,8 +34,14 @@ class Evented(MacroElement):
     """
 
     def on(self, **event_map: JsCode):
+        self._add(False, **event_map)
```

Comment:
I think it's nicer to use keyword arguments when the value doesn't inform what argument it's applying to.

```suggestion
        self._add(once=False, **event_map)
```